src/api/blog.ts

```diff
@@ -25,3 +28,9 @@ export function postBlog(id: string | null, data: any) {
   const URL = endpoints.admin.blogs.blog(id);
   return postFetcher([URL, {}, data]);
 }
+
+export function storageBlog(file: File, endpoint?: string) {
+  const address = file.name + uuid();
+  const _ref = ref(storage, `${endpoints.admin.storage.blogImage + address}`);
```

Comment:
can you make this function more generic, call it fileStorage or postFile or something and have the endpoint be a function where you can pass the bucket / folder name into to generate that specific route